genome_designer/genome_finish/tests/test_genome_finish.py

```diff
@@ -290,30 +300,106 @@ def _run_contig_walk_test(self, test_dir):
+            new_ref_genome_params = {'label': 'new_ref'}
+            new_ref_genome = generate_new_reference_genome(
+                    variant_set, new_ref_genome_params)
+
+            # Verify insertion was placed correctly
+            new_ref_genome_fasta = get_dataset_with_type(
+                    new_ref_genome, Dataset.TYPE.REFERENCE_GENOME_FASTA
+                    ).get_absolute_location()
+
+            fastas_same, indexes = are_fastas_same(
+                    self.target_fasta, new_ref_genome_fasta)
+
+            self.assertTrue(fastas_same)
+
+        else:
+            print 'Multiple variants'
+            incorrect_variants = []
+            for i, v in enumerate(variant_set.variants.all()):
+
+                # Make Variant set for single variant
+                variant_set = VariantSet.objects.create(
+                        reference_genome=self.reference_genome,
+                        label='multiple_variant_%d' % i)
+
+                update_variant_in_set_memberships(
+                    self.reference_genome,
+                    [v.uid],
+                    'add',
+                    variant_set.uid)
+
+                new_ref_genome_params = {'label': 'new_ref'}
+
+                new_ref_genome = generate_new_reference_genome(
+                        variant_set, new_ref_genome_params)
+
+                # Verify insertion was placed correctly
+                new_ref_genome_fasta = get_dataset_with_type(
+                        new_ref_genome, Dataset.TYPE.REFERENCE_GENOME_FASTA
+                        ).get_absolute_location()
+
+                fastas_same, indexes = are_fastas_same(
+                        self.target_fasta, new_ref_genome_fasta)
+
+                if not fastas_same:
+                    incorrect_variants.append((i, v))
+
+            if incorrect_variants:
+                print 'Variant %d/%d was incorrect' * len(
+                        incorrect_variants) * tuple(zip(
+                                [t[0] for t in incorrect_variants],
+                                len(incorrect_variants) *
+                                len(variant_set.variants.all())))
+            self.assertTrue(not incorrect_variants, '%d incorrect variants' % (
+                    len(incorrect_variants) /
+                    len(variant_set.variants.all())))
 
     def test_deletion(self):
         test_dir = os.path.join(GF_TEST_DIR, 'random_seq_data',
                 'deletion')
-        self._run_contig_walk_test(test_dir)
+        variant_set = self._run_contig_walk_test(test_dir)
+        self._assert_variants_make_target(variant_set)
 
     def test_homology_flanked_deletion(self):
         test_dir = os.path.join(GF_TEST_DIR, 'random_seq_data',
                 'homology_flanked_deletion')
-        self._run_contig_walk_test(test_dir)
+        variant_set = self._run_contig_walk_test(test_dir)
+        self._assert_variants_make_target(variant_set)
+
+    def test_is_element(self):
```

Comment:
Love the tests.

Can you clarify the name of this test or add a comment? 'is' is not immediately obvious to refer to 'insertion element'.